model/installation_request.go

```diff
@@ -460,9 +462,12 @@ func (p *PatchInstallationRequest) addMissingIngressSourceRanges(installation *I
 	}
 
 	allowedIPRanges := strings.Split(installation.AllowedIPRanges, ",")
+
 	for _, ip := range ips {
-		if !contains(allowedIPRanges, ip) {
+		if IsIPRangeValid(ip) && !contains(allowedIPRanges, ip) {
 			allowedIPRanges = append(allowedIPRanges, ip)
+		} else if !IsIPRangeValid(ip) {
+			logrus.Info("Invalid IP range will not be appended: ", ip)
```

Comment:
I think the same thing applies here. For patching we should enforce the request to be valid.

Reply:
I'm +1 on an error, and think we should not just throw out invalid ranges. From a user perspective (both myself the consuming dev and the end users) it'll be better to let people know there's an issue with their config than to silently throw out anything that's invalid. To the end user, disappearing config would look buggy